akka-http-core/src/main/resources/reference.conf

```diff
@@ -404,4 +404,6 @@ akka.http {
     # HttpResponse on client side).
     tls-session-info-header = off
   }
+
+  strict-entity-max-bytes = 9223372036854775807
```

Comment:
Yeah, I think so since we also want to be compatible with previous 10.x releases. 
So it should not start blowing up once people upgrade to 10.0.2.
We could decrease it in 10.1 perhaps.